phpmd.xml

```diff
@@ -10,7 +10,12 @@
     </rule>
     <rule ref="rulesets/codesize.xml/TooManyMethods">
         <properties>
-            <property name="maxmethods" value="20" />
+            <property name="maxmethods" value="16" />
```

Comment:
I think we discussed this. In a small component like this it makes sense to lower such limits to the smallest number possible. If we ever want to add code that needs a slightly higher limit it's very easy (at least I hope it is) to increase this number in the same patch. It becomes part of the review then.